src/tools/rbd/action/DiskUsage.cc

```diff
@@ -206,6 +208,10 @@ static int do_disk_usage(librbd::RBD &rbd, librados::IoCtx &io_ctx,
       ++count;
     }
   }
+  if (!found) {
+    std::cerr << "specified image of " << imgname << "is not found." << std::endl;
```

Comment:
@dillaman this is already updated, but I am not sure why it's still expanded here in github.